complex/ft_main.c

```diff
@@ -471,6 +472,8 @@ static void ft_fw_usage(char *program)
 	FT_PRINT_OPTS_USAGE("-x", "exit after test run");
 	fprintf(stderr, "\nClient only options:\n");
 	FT_PRINT_OPTS_USAGE("-f <test_config_file>", "");
+	FT_PRINT_OPTS_USAGE("-a <provider_name>", "");
+	FT_PRINT_OPTS_USAGE("-t <test_name>", "");
```

Comment:
Suggestion: can we use 'u' option for ubertest config file and keep 'f' for provider name? We can use the 'u' option for providing config file name in runfabtests too.